src/main/java/taxi/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,57 @@
+package taxi.dao.impl;
+
+import java.util.List;
+import java.util.Optional;
+import taxi.dao.ManufacturerDao;
+import taxi.db.Storage;
+import taxi.exception.NoSuchManufacturerException;
+import taxi.lib.Dao;
+import taxi.model.Manufacturer;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        Storage.addManufacturer(manufacturer);
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        return Storage.manufacturers
+                .stream()
+                .filter(man -> man.getId().equals(id))
```

Comment:
```suggestion
                .filter(m -> m.getId().equals(id))
```
this word translation may confuse when reading the code

Reply:
refactored, now the context looks clearer. Thank you